portal/templates/profile_macros.html

```diff
@@ -714,89 +714,119 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
+                                if (oo.children.length > 0) {
+                                    contentHTML = "<legend orgId='%orgId%'>%translatedOrgName%</legend>";
+                                    contentHTML += "<input class='tnth-hide' type='checkbox' name='organization' parent_org='true' org_name='%orgName%' ";
+                                    contentHTML += " id='%orgId%_org' value='%orgId%' />";
+                                    contentHTML = contentHTML.replace(/\%orgId\%/g, item.id)
+                                                            .replace(/\%orgName\%/g, item.name)
+                                                            .replace(/\%translatedOrgName\%/g, i18next.t(item.name));
+                                    
+                                    //contentHTML = "<legend orgId='" + item.id + "'>"+ i18next.t(item.name) +"</legend><input class='tnth-hide' type='checkbox' name='organization' parent_org='true' org_name='" + item.name + "' id='" + item.id + "_org' value='"+item.id+"' />";
+                                    $("#" + state + "_container").append(contentHTML);
+                                };
+                            };
+                        };
+                    });
+                
+                    /**** draw input element(s) that belongs to each state based on parent organization id ***/
+                    entry.forEach(function(item) {
+                        if (parseInt(item.id) != 0 && item.partOf) {
+                            var parentId = (item.partOf.reference).split("/")[2];
+                            if (parentId) {
+                                var parentState = (function(o) {
+                                    var s = "", found = false;
+                                    for (var state in states) {
+                                        if (!found) {
+                                            (states[state]).forEach(function(i) {
+                                                if (i == o) {
+                                                   s = state;
+                                                   found = true;
+                                                };
+                                            });
+                                        };
+                                    };
+                                    return s;
+                                })(parentId);
+
+                                contentHTML = "<div class='radio'>" +
+                                            "<label><input class='clinic' type='radio' id='%orgId%_org' value='%orgId%' state='%state%' name='organization' data-parent-name='%parentOrgName%' data-parent-id='%parentOrgId%'>%translatedOrgName%</label>" +
+                                            "</div>";
+                                contentHTML = contentHTML.replace(/\%orgId\%/g, item.id)
```

Comment:
Could you use the more standard `{{var}}` or `{var}`? Characters that match in sets are a little easier to read and grep for.